Source/com/drew/metadata/ico/IcoReader.java

```diff
@@ -34,10 +34,8 @@
  *
  * @author Drew Noakes https://drewnoakes.com
  */
-public class IcoReader
-{
-    public void extract(@NotNull final SequentialReader reader, @NotNull final Metadata metadata)
-    {
+public class IcoReader {
+    public void extract(@NotNull final SequentialReader reader, @NotNull final Metadata metadata) {
```

Comment:
Please revert the formatting change.

Reply:
Sorry, my IDE does that. It is reverted now